### DISCUSSION TITLE
[Suggestion] Toast position

### DISCUSSION THREAD

**Discussion by dudelisius** [livewire/flux#148] [Feature Requests] [answered]
It would be nice if we could set the position of the toast, I'm thinking of these options;

Top left 
Top center
Top right
Bottom left
Bottom center
Bottom right

**Comment by WorthLMS** [comment]
Agreed. I was doing my own setup for position, icon and variant would be nice to be able to have this setup. Could also be useful to have a persist prop that wraps it in @persist automagically

**Comment by joshhanley** [comment]
Thanks for reporting! We have converted this issue to a feature request for now, so we can focus on fixing any bugs first.

**Comment by dudelisius** [comment, marked as answer]
I noticed in the [latest release](https://github.com/livewire/flux/releases/tag/v1.0.9) that both [variants](https://fluxui.dev/components/toast#variants) and [positions](https://fluxui.dev/components/toast#positioning) were added. Therefore closing this suggestion.

Thanks @calebporzio !

**Reply by donnyxray** [reply]
It looks like we have to specify the location each and every time - am I wrong? Typically the toast location is consistent throughout an application. I would've expected to set it in a config file.

**Reply by dudelisius** [reply]
That is correct. I solved this by creating a small wrapper for the Toast that I call from several places within my application. This way I only need to set the position settings in one place.

**Comment by DaW-S** [comment]
I appreciate the positioning but I'm still missing center option. 
I'd really like to use top center or bottom center but it seems there is no such option.

**Reply by axeldotdev** [reply]
I needed to so I published the component and added `[&[data-position*=center]]:mx-auto` in the classes. The customisation of the components is really simple. 👍

**Reply by DaW-S** [reply]
Works great, thank you very much.
